Model/Cron.php

```diff
@@ -1248,18 +1249,19 @@ protected function _isAutoCapture()
                 );
                 return true;
             }
+
             // if PayPal capture modues is different from the default use this one
-            if (strcmp($this->_paymentMethod, 'paypal') === 0 && $captureModePayPal != "") {
-                if (strcmp($captureModePayPal, 'auto') === 0) {
-                    $this->_adyenLogger->addAdyenNotificationCronjob(
-                        'This payment method is paypal and configured to work as auto capture'
-                    );
-                    return true;
-                } elseif (strcmp($captureModePayPal, 'manual') === 0) {
+            if (strcmp($this->_paymentMethod, 'paypal') === 0) {
+                if ($captureModePayPal) {
```

Comment:
good one, I would also suggest to change the variable name to a less misleading one (e.g. $manualCapturePayPal) 
..and maybe return !$manualCapturePayPal in both cases